srf_generation/input_file_generation/generate_srf_from_realisations.py

```diff
@@ -39,16 +74,34 @@ def process_common_realisation_file(cybershake_root, realisation_file):
     if realisation["type"] == 1:
         create_info_file(
             srf_file=srf_file,
-            srf_type=realisation["type"],
-            mag=realisation["magnitude"],
-            rake=realisation["rake"],
+            srf_type=realisation.get("type"),
```

Comment:
Why's this one hardcoded but the below one isn't?

Is there a SRF type enum?

Reply:
No enum, as the types are literally just 1-4 I don't think it would help, unless we gave the types proper names
Changed to be hard coded like the one below as it must be the type found in the switch statement

Reply:
They kinda do have names, but 2 and 3 are a bit contrived to explain.

Reply:
I think it's a bit silly as 1&2 are from gcmt, 3&4 are from nhm (normally, you could probably get a type 4 from gcmt)

Reply:
Yeah, when we get to the larger gcmt quakes we'll have type 4 definitions for them. (hardcoded for each quake)

Reply:
Though all nhm generated faults are currently type 4.

Reply:
Currently there isn't a researcher working on manually generated SRFs but it does happen.